src/routes/jira/server/jira-edit-app-get.ts

```diff
@@ -0,0 +1,24 @@
+import { NextFunction, Request, Response } from "express";
+import { GitHubServerApp } from "models/github-server-app";
+
+export const JiraEditAppGet = async (
+	req: Request,
+	res: Response,
+	next: NextFunction
+): Promise<void> => {
+	try {
+		req.log.debug("Received Jira edit app page request");
+
+		const gitHubServerAppId = req.params.id as string;
+		const app = await GitHubServerApp.getForGitHubServerAppId(parseInt(gitHubServerAppId));
```

Comment:
@mboudreau , so there's line here for looks for `GitHubServerApp`. Tagging you because we had a chat this morning. Do you think we should preserve an app instance in `res.locals.gitHubServerApp`? So that we don't need to call to db multi times. Any route under `/github` will most likely need it anyway.

As for the code inconsistency, I believer only place will be when you create the `gitHubAppClient`, which is only few places.

Reply:
And it is NOT related to this PR, sorry K, closing this comment now.